utils/minor_data_modifications.py

```diff
@@ -0,0 +1,121 @@
+# This script can be used to either add or remove input EOS symbols, and add attentive guidance indices
```

Comment:
why is this a script instead of a function. Did you write it ?

Reply:
Nope it was written by someone else, didn't know if I could remove it completely yet..

Reply:
Not entirely done reworking the PRs but about this comment: I've opted to remove the script completely to clean it up.